app/layout.tsx

```diff
@@ -18,8 +19,11 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
-      <Footer />
+      <body className={inter.className}>
+        <Navbar />
+        {children}
```

Comment:
I've been thinking about this a lot.... Do we want to put a `<main>` element in the `layout.tsx` or do we want each page to be responsible for it's own `<main>` and accompanying styling?

You have a system that's working for you, but I was feeling like I wanted `<main>` here with styles (depending on whatever kind of layout mode we wanted) and then have each page define only content inside the `<main>`. I haven't been... perfectly consistent about that in the RIDDC project, but. I'm curious to hear if you have thoughts.